README.md

```diff
@@ -5,91 +5,114 @@
 [![Discord](https://img.shields.io/discord/453243919774253079.svg)](https://discord.gg/tsBTgc9)
 
 SourceCred creates reputation networks for open-source projects.
-Any open-source project can create its own _cred_, which is a reputational metric showing how much credit contributors deserve for helping the project.
-To compute cred, we organize a project’s contributions into a graph, whose edges connect contributions to each other and to contributors.
-We then run PageRank on that graph.
+Any open-source project can create its own _cred_, which is a
```

Comment:
Please don’t reformat the rest of the file with unrelated changes; this
kills the blame. It looks like `README.md` uses one-sentence-per-line
wrapping, so within this file your additions should do the same and your
changes shouldn’t reflow other lines.

(The purpose of one-sentence-per-line is made editing and diffing
easier; reflowing breaks that.)

Reply:
So should `README.md` not  adhere to the 72-character wrap then? I'm a little confused. The reason I made these changes was because I thought both files needed to be limited in width and I noticed that a number of lines in `README.md` fell outside that constraint.

Reply:
The important thing is to be consistent with the existing formatting of
the files that you edit. Occasionally, these standards are themselves
inconsistent across files. Some of our shell scripts have 2-space
indents; some have 4-space indents. The `CONTRIBUTING.md` file
hard-wraps at column 72, while `README.md` wraps at the end of every
sentence instead of a fixed column. This may not be ideal, but it’s
better to remain locally consistent and accept the global inconsistency
than it is to reblame the entire file on an unrelated change.

We would only make these files globally consistent if we also configured
a lint rule to verify this prior to merging any PRs, and preferably an
autoformatter so we don’t have to think about this.

So: `README.md` should indeed not adhere to the 72-character wrapping
found in `CONTRIBUTING.md`.

Reply:
Awesome, thank you for the clarification. I'll be sure to pay attention to these details going forward.

Reply:
Thanks, and sorry for the confusion—having to explain this at all is a
big part of the reason that we love autoformatters. :-)